Update manage-mail-flow-using-third-party-cloud.md (#1998)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mail-flow-best-practices/manage-mail-flow-using-third-party-cloud.md

> [!NOTE]
> If you already have an **OnPremises** inbound connector for the same certificate or sender IP addresses, you still need to create the  **Partner** inbound connector (the *RestrictDomainsToCertificate* and *RestrictDomainsToIPAddresses* parameters are only applied to **Partner** connectors). The two connectors can coexist without problems. 

> There are two options for the next step that you might decide; if you want to prevent double anti-spam check, you may choose option A, this will prevent most of EOP and Office 365 ATP controls through a bypass transport rule. If you want to take advantage of EOP and Office 365 ATP controls, you should opt for option B which consist of enabling Enhanced Filtering.

Reviewer: ```suggestion
There are two options for the next step that you may choose from; if you want to prevent double anti-spam check, you may choose option A, which will prevent most of EOP and Office 365 ATP controls through a bypass transport rule. If you want to take advantage of EOP and Office 365 ATP controls you should opt for option B, which consists of enabling Enhanced Filtering.
```

> [!IMPORTANT]

> > In alternative of bypass spam filtering through transport rule, we recommend you enable [Enhanced Filtering Connector (also known as Skip Listing)](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors). Most of times third-party cloud anti-spam shares IPs across many customers, bypassing IPs in this situation may result in spoofed and phishing messages being marked as legitimate from these IPs.

Reviewer: ```suggestion
> As an alternative to bypassing spam filtering using a transport rule, we recommend you enable [Enhanced Filtering Connector (also known as Skip Listing)](https://docs.microsoft.com/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors). Most of the time third-party cloud anti-spam shares IPs across many customers. Bypassing IPs in this situation may result in spoofed and phishing messages being marked as legitimate from these IPs.
```

#### Option A:

> Create a mail flow rule (also known as a transport rule) in the Exchange admin center (EAC) at **Exchange admin > Mail flow > Rules** to bypass anti-spam check:

Reviewer: ```suggestion
Create a mail flow rule (also known as a transport rule) in the Exchange admin center (EAC) at **Exchange admin** > **Mail flow** > **Rules** to bypass the anti-spam check:
```

![Mail flow rule to prevent double-scanning](../media/TransportRuleFor3rdParty.png)

#### Option B (recommended):

> Enable [Enhanced Filtering](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors#procedures-for-enhanced-filtering-for-connectors) on the Inbound Connector of type *Partner*, which you created to receive messages from the third-party application.

Reviewer: ```suggestion
Enable [Enhanced Filtering](https://docs.microsoft.com/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors#procedures-for-enhanced-filtering-for-connectors) on the Inbound Connector of the *Partner* type that you created to receive messages from the third-party application.
```

> [!NOTE]

> > For hybrid scenarios where third-party application relay on Exchange on-premises to send to EXO, you need to enable Enhanced Filtering on the Inbound Connector of type *OnPremises*. Even so, you can lockdown inbound flow using Inbound Connector of type *Partner*.

Reviewer: ```suggestion
> For hybrid scenarios where third-party applications rely on Exchange on-premises to send to EXO, you need to enable Enhanced Filtering on the Inbound Connector type *OnPremises*. Even so, you can lock down inbound flow using Inbound Connector type *Partner*.
```

### Scenario 2 - MX record points to third-party solution without spam filtering

I plan to use Exchange Online to host all my organization's mailboxes. All email that's sent to my domain from the internet must first flow through a third-party archiving or auditing service before arriving in Exchange Online. All outbound email that's sent from my Exchange Online organization to the internet must also flow through the service. However, the service doesn't provide a spam filtering solution.

> We can support this scenario only if you enabled Enhanced Filtering. If you choose to bypass inbound flow through transport rule, we cannot support this scenario because the inbound mail flow through the service causes Office 365 spam and phish filtering to not work properly (mail from all internet senders appears to originate from the third-party service, not the true email source on the internet). 

Reviewer: ```suggestion
We can support this scenario only if you enabled Enhanced Filtering. If you choose to bypass inbound flow through a transport rule, we cannot support this scenario because the inbound mail flow through the service causes Office 365 spam and phish filtering to not work properly (mail from all internet senders appears to originate from the third-party service, not the true email source on the internet). 
```

I plan to use Exchange Online to host all my organization's mailboxes. All email that's sent to my domain from the internet must first flow through a third-party archiving or auditing service before arriving in Exchange Online. All outbound email that's sent from my Exchange Online organization to the internet must also flow through the service. However, the service doesn't provide a spam filtering solution.

If you choose to bypass inbound flow through transport rule, we cannot support this scenario because the inbound mail flow through the service causes Office 365 spam and phish filtering to not work properly (mail from all internet senders appears to originate from the third-party service, not the true email source on the internet). 
If you choose to enable Enhanced Filtering, your organization's mail flow setup looks like the following diagram:

![Mail flow diagram showing inbound mail from the internet to a third-party solution to Office 365 and outbound mail from Office 365 to the third-party solution to the internet.](../media/05300b2e-1223-4eb2-87df-b3370fac9f91_2.png)

#### Best practices for using a third-party cloud service with Office 365

> Although, it is possible to keep Enhanced Filtering in this scenario, it is not recommended. We strongly recommend that you use the archiving and auditing solutions that are provided by Office 365

Reviewer: ```suggestion
Although it is possible to keep Enhanced Filtering in this scenario, it is not recommended. We strongly recommend that you use the archiving and auditing solutions that are provided by Office 365
```
